src/Helpers/auth_helper.php

```diff
@@ -43,3 +44,33 @@ function user_id()
         return $auth->id();
     }
 }
+
+if (! function_exists('shieldSetting')) {
+    /**
+     * Provides a wrapper function for settings module.
+     *
+     * @param mixed $value
+     *
+     * @return array|bool|float|int|object|string|void|null
+     * @phpstan-return ($value is null ? array|bool|float|int|object|string|null : void)
```

Comment:
In what instance will this return void?

```suggestion
     * @return mixed
```